docs/versioning_integration.md

```diff
@@ -0,0 +1,60 @@
+Versioning is an optional feature, so core CMS and 3rd-party apps
+should be still usable when versioning is not enabled.
+
+# Integrating Versioning
+
+Let's use an example. There's an existing `blog` application
+that has a single model called `Post`:
+
+```python
+# polls/models.py
+class Post(models.Model):
+    title = models.CharField(max_length=100)
+    text = models.TextField()
+```
+
+Versioning works by creating multiple objects (called **content objects**).
+To know what these objects are versions of, there's a contept of **grouper objects**,
+which tie select content objects together.
+
+In order to make `blog` app work with versioning, changes to model structure are needed.
+
+```python
+# polls/models.py
+class Post(models.Model):
+    pass
+
+
+class PostContent(models.Model):
+    post = models.ForeignKey(Post, on_delete=models.CASCADE)
+    title = models.CharField(max_length=100)
+    text = models.TextField()
+```
+
+This structure is not only compatible with versioning,
+but also works when versioning is not enabled.
```

Comment:
Is there supposed to be an explanation for when versioning is not enabled for example, what happens to ```Post``` and ```PostContent``` ?

Reply:
Addressed